src/languageHandler.tsx

```diff
@@ -144,6 +153,11 @@ export function _t(text: string, variables?: IVariables, tags?: Tags): string |
     }
 }
 
+export function _tSasV1(emoji: string): string | React.ReactNode {
```

Comment:
this should only ever return a string as we don't pass options though

Reply:
Unless a developer changes the constant to have it return a node

Reply:
The definition of `_t` in the usage you have says `string`

`export function _t(text: string, variables?: IVariables): string;`

Reply:
hmm, alright. The types seem incorrect but trying to thread the code path through my head isn't working either.